frontend/src/pages/Plant/index.tsx

```diff
@@ -43,20 +50,35 @@ const PlantPage = () => {
 		const updatedPlantData: NewPlant = {
 			name: plant!.name,
 			species: plant!.species,
-			// imageUrl: plant!.imageUrl,
-			uploadedImage: plant!.uploadedImage,
+			imageUrl: plant!.imageUrl,
+			// uploadedImage: plant!.uploadedImage,
 			wateringFrequencyInDays: plant!.wateringFrequencyInDays,
 			lastWatered: getTodaysDate(),
 			waterAmountInMl: plant!.waterAmountInMl,
-			imageUrl: plant!.imageUrl,
 			location: plant!.location,
-			lastWateredBy: plant!.lastWateredBy
+			lastWateredBy: plant!.lastWateredBy,
 		};
-		const response = await updatePlant(plant!.id, householdId!, updatedPlantData);
+		const response = await updatePlant(
+			plant!.id,
+			householdId!,
+			updatedPlantData
+		);
 		toast.success(`${plant?.name} has been successfully watered`);
 		setPlant(response);
 	};
 
+	const viewLoadingSkeleton = () => {
+		return (
+			<div className='flex flex-col pl-8 gap-4 w-full'>
+				<div className='skeleton h-80 w-80 pl-12'></div>
+				<div className='skeleton h-16 w-44 pl-12'></div>
+				{[1, 2, 3, 4].map((item) => (
+					<div key={item} className='skeleton w-44 h-8 bg-base-200'></div>
```

Comment:
I experimented with the width and i think w-60 looks more natural bcs rn the width is the same as the title skeleton

Reply:
done :)